validmind/tests/run.py

```diff
@@ -83,32 +83,39 @@ def combine_tables(table_index):
     )
 
 
-def _update_plotly_titles(figures, input_groups, title_template):
-    current_title = figures[0].figure.layout.title.text
+def _update_plotly_titles(figures, input_group, title_template):
+    for figure in figures:
+
+        current_title = figure.figure.layout.title.text
+
+        input_description = " and ".join(
+            f"{k}: {v if isinstance(v, str) else ', '.join(item.input_id for item in v) if isinstance(v, list) and all(hasattr(item, 'input_id') for item in v) else v.input_id}"
```

Comment:
Can you run `make format` to get this line to format properly? I think I wrote it so I take full responsibility 😅 but its really hard to read now that I am coming back to it.